source/localizable/tutorial/routes-and-templates.md

```diff
@@ -194,22 +219,40 @@ Unlike the other route handlers we've made so far, the `index` route is special:
 it does NOT require an entry in the router's mapping.
 We'll learn more about why the entry isn't required when we look at nested routes in Ember.
 
-Let's update our `index.hbs` with some HTML for our home page and our links to the other routes in our application:
+We can start by implementing the unit test for index.
+Since all we want to do is transition to rentals our unit test will just verify that the route's `transitionTo` method is called with the desired route.
+We'll verify this by stubbing the `transitionTo` method for the route and asserting that the `rentals.index` route is passed when called.
 
-```app/templates/index.hbs
-<div class="jumbo">
-  <div class="right tomster"></div>
-  <h2>Welcome!</h2>
-  <p>
-    We hope you find exactly what you're looking for in a place to stay.
-    <br>Browse our listings, or use the search box above to narrow your search.
-  </p>
-  {{#link-to 'about' class="button"}}
-    About Us
-  {{/link-to}}
-</div>
+```tests/unit/routes/index-test.js
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:index', 'Unit | Route | index');
+
+test('should transition to rentals route', function(assert) {
+  let route = this.subject({
+    transitionTo(routeName) {
+      assert.equal(routeName, 'rentals.index', 'transition to route name rentals.index');
+    }
+  });
+  route.beforeModel();
+});
+```
+
+In our index route, we simply add the `transitionTo` invocation.
+
+```app/routes/index.js
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  beforeModel() {
+    this._super(...arguments);
+    this.transitionTo('rentals.index');
+  }
+});
```

Comment:
Hmm, not sure I'm a fan of doing things this way.  It makes it clearer to someone working with Ember for the first time (so I like that) but do we want to be adding in `transitionTo` calls in a `beforeModel` hook to our tutorial?  Seems like we could alternately map rentals to the path of `rentals.index` like the following:

```js
this.route('rentals', { path: '/' });
```

@locks do you have thoughts here?